Scripts/Tracking/Follow/Modifier/Property/Scale/TransformScale.cs

```diff
@@ -22,13 +16,14 @@ public class TransformScale : PropertyModifier
         /// <param name="offset">The offset of the target against the source when modifying.</param>
         protected override void DoModify(GameObject source, GameObject target, GameObject offset = null)
         {
-            if (useLocalScale)
+            if (offset == null)
             {
-                target.transform.localScale = (offset != null ? (source.transform.localScale - (offset.transform.localScale - target.transform.localScale)) : source.transform.localScale);
+                target.transform.SetGlobalScale(source.transform.lossyScale);
             }
             else
             {
-                target.transform.SetGlobalScale((offset != null ? (source.transform.lossyScale - (offset.transform.lossyScale - target.transform.lossyScale)) : source.transform.lossyScale));
+                Vector3 offsetMultiplier = Vector3Extensions.Divide(1, offset.transform.localScale);
+                target.transform.SetGlobalScale(Vector3.Scale(source.transform.lossyScale, offsetMultiplier));

```

Comment:
```suggestion
                target.transform.SetGlobalScale(source.transform.lossyScale.Divide(offset.transform.localScale));
```

Reply:
I can't see what the suggested change is here because github is making it look all confusing.

are you saying instead of: 
```
Vector3 offsetMultiplier = Vector3Extensions.Divide(1, offset.transform.localScale);
target.transform.SetGlobalScale(Vector3.Scale(source.transform.lossyScale, offsetMultiplier));
```

to just do

```
target.transform.SetGlobalScale(source.transform.lossyScale.Divide(offset.transform.localScale));
```

If so, I think that causes issues where you can get divisions by zero. The previous way avoids divisions by zero (unless you actually put a 0 in for the local scale)

Reply:
My suggestion is to not have the variable above (see the suggestion deleting the whole line above this) because you introduced the extension method doing the exact same thing.

You already do `1f / vector` above, I didn't change any logic. I can see how that division can fail when dividing by zero, though.
I wouldn't worry about it as that would mean someone has set the local scale to zero, which is not really valid anyhow.